ci/run_ctests.sh

```diff
@@ -6,5 +6,16 @@ set -euo pipefail
 # Support customizing the ctests' install location
 cd "${INSTALL_PREFIX:-${CONDA_PREFIX:-/usr}}/bin/tests/libkvikio/"
 
-# Run BASIC_IO_TEST
-./BASIC_IO_TEST
+# Run basic tests
+rapids-logger "Run BASIC_IO_EXAMPLE"
+./BASIC_IO_EXAMPLE
+rapids-logger "Run BASIC_NO_CUDA_EXAMPLE"
+./BASIC_NO_CUDA_EXAMPLE
+
+# Run gtests
+rapids-logger "Run gtests"
+./cpp_tests
+# TODO: how to use ctest instead of executing the test directly?
+# The following line fails with a "ctest doesn't exist" in CI.
+# Do we need to install ctest in the CI images?
+# ctest --no-tests=error --output-on-failure "$@"
```

Comment:
Do we need to install `ctest`?

Reply:
You need to install cmake. Add to the test_cpp group.